### DISCUSSION TITLE
implementation of select

### DISCUSSION THREAD

**Discussion by grzesiekb** [joaoh82/rust_sqlite#5] [General]
Executing a simple select statement from table
statement so that it returns the value of.

![select](https://user-images.githubusercontent.com/1280575/111900219-8d161f80-8a31-11eb-9ed3-6a1f1daac176.jpg)

**Comment by joaoh82** [comment]
Thanks for this. Yeah, got back to work on it this weekend.

![SQLRite Simple SQL Execution High Level Diagram](https://user-images.githubusercontent.com/208517/111904035-0ff4a580-8a45-11eb-9a12-d9f626087394.png)
